self-guided-lessons/htcondor-more-options.md

```diff
@@ -0,0 +1,209 @@
+# More submit file options
+
+{% objectives "Learning Objectives" %}
+* Learn some more options you can specify for your jobs
+* Be able to submit jobs with input/output files and arguments
+* Be able to debug some common errors
+{% endobjectives %} 
+
+The options showcased in submit file used in the previous section are enough for certain simpler jobs, but for many purposes you will want to do more. This section of the lesson will cover a selection of some of the more common and/or useful ones, and the settings you'll likely want to use for your own jobs. For more information, a complete list can be found [in the documentation](https://research.cs.wisc.edu/htcondor/manual/v7.6/condor_submit.html) for `condor_submit`.
+
+### Input files
+
+Often, you'll want your job to process data that's stored in another file. Create a file `input.txt` and add some lines of text to it. To test this, you can use the following bash script, placed in `exec.sh`:
+
+```
+#!/bin/bash
+echo "# Contents of file 'input.txt': #"
+cat input.txt
+```
+
+Remember to make the script exectable, and to test it locally first. Then, go into your submit file. Here, you'll want to make sure that the `executable` option is set to run `exec.sh`, and also to add the following line to the body of your submit file:
+
+```
+transfer_input_files = input.txt
```

Comment:
Does it matter where in the file this goes, e.g. before/after `queue`?

Reply:
I haven't confirmed this, but I think it should be before `queue` (at least that's what I've done in all of my scripts, and seen in all examples). I've added a clarification of this, thanks.